spec/schemas/_core.get.yaml

```diff
@@ -9,6 +9,9 @@ components:
     GetResult:
       type: object
       properties:
+        _type:
+          $ref: '_common.yaml#/components/schemas/Type'
+          x-version-removed: '2.0'
```

Comment:
This is redundant since `_common.yaml#/components/schemas/Type` already has this property.

Reply:
What's the right place for it? In `components/schemas/Type` or here?

Reply:
The entire concept of doc type was removed in 2.0 so putting it in `components/schemas/Type` makes the most sense